src/rgw/services/svc_bi.h

```diff
@@ -29,7 +29,7 @@ class RGWSI_BucketIndex : public RGWServiceInstance
   RGWSI_BucketIndex(CephContext *cct) : RGWServiceInstance(cct) {}
   virtual ~RGWSI_BucketIndex() {}
 
-  virtual int init_index(RGWBucketInfo& bucket_info) = 0;
+  virtual int init_index(RGWBucketInfo& bucket_info, const Span& parent_span = nullptr) = 0;
```

Comment:
I would like to see a namespace qualified (not sure.., e.g.) jaeger_tracing::Span rather than namespace-lifted Span, to avoid namespace clutter, and to make it easy to find it

Reply:
noted